stestr/config_file.py

```diff
@@ -83,11 +85,17 @@ def get_run_command(self, test_ids=None, regexes=None,
             to use for the run.
         :param int concurrency: How many processes to use. The default (0)
             autodetects your CPU count and uses that.
-        :param str blacklist_file: Path to a blacklist file, this file contains
-            a separate regex exclude on each newline.
-        :param str whitelist_file: Path to a whitelist file, this file contains
-            a separate regex on each newline.
-        :param str black_regex: Test rejection regex. If a test cases name
+        :param str blacklist_file: DEPRECATED: soon to be replaced by the new
+            option exclusion_list_file below.
+        :param str exclusion_list_file: Path to an exclusion list file, this
+            file contains a separate regex exclude on each newline.
+        :param str whitelist_file: DEPRECATED: soon to be replaced by the new
+            option inclusion_list_file below.
+        :param str inclusion_list_file: Path to an inclusion list file, this
+            file contains a separate regex on each newline.
+        :param str black_regex: DEPRECATED: soon to be replaced by the new
+            option exclusion_regex below.
```

Comment:
I think you can remove the deprecated args here, we don't treat `TestrConf` as a Stable API. See https://stestr.readthedocs.io/en/latest/api.html#internal-apis

Reply:
Changing "deprecation note" to simple mention of new args...